.github/workflows/build-simple-docsite.yml

```diff
@@ -0,0 +1,57 @@
+# This workflow will build a simple Sphinx-based docsite
+
+name: Build simple docsite
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+  # Run once per week (Monday at 06:00 UTC)
+  schedule:
+    - cron: '0 6 * * 1'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.9
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install . coverage codecov
+
+      - name: Use antsibull-docs sphinx-init
+        run: |
+          coverage run -p --source antsibull -m antsibull.cli.antsibull_docs sphinx-init --use-current --lenient --dest-dir .
+
+      - name: Patch build.sh to supply code coverage
+        run: |
+          sed -i build.sh -e 's/antsibull-docs /coverage run -p --source antsibull -m antsibull.cli.antsibull_docs /g'
+          sed -i build.sh -e 's/sphinx-build /coverage run -p --source antsibull --source sphinx_antsibull_ext -m sphinx.cmd.build /g'
+
+      - name: Install dependencies
+        run: |
+          pip install ansible-core
+          pip install -r requirements.txt
+
+      - name: Install collections
+        run:
+          ansible-galaxy collection install community.docker community.crypto
+
+      - name: Build docsite
+        run: |
+          ./build.sh
+
+      - name: Combine and upload coverage stats
+        run: |
+          coverage combine .coverage.*
+          coverage report
+          coverage xml -i
+          codecov --token=${{ secrets.CODECOV_TOKEN }}
```

Comment:
codecov does not need tokens in public GH repos

Reply:
Fixed in 46cb2f524ea0f90fbdb8dad19ed4d63373e4d343.